File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020806Activity.java
package com.liushu.crazyandroid.ui.stage01.chapter02.activity;

import android.widget.ImageView;
import android.widget.TextView;

import com.jaydenxiao.common.base.BaseActivity;
import com.liushu.crazyandroid.R;

import butterknife.BindView;
import butterknife.OnClick;

public class Demo020806Activity extends BaseActivity {

    @BindView(R.id.iv_back)
    ImageView mIvBack;
    @BindView(R.id.tv_title)
    TextView mTvTitle;

    @Override
    public int getLayoutId() {
        return R.layout.activity_demo020806;
    }

    @Override
    public void initView() {
        mTvTitle.setText("可垂直和水平滚动的视图");
    }

    @OnClick(R.id.iv_back)
    public void onViewClicked() {
        finish();
    }
}

File: app/src/main/java/com/liushu/crazyandroid/ui/stage01/chapter02/activity/Demo020806Activity.kt
package com.liushu.crazyandroid.ui.stage01.chapter02.activity

import butterknife.OnClick
import com.jaydenxiao.common.base.BaseActivity
import com.liushu.crazyandroid.R
import kotlinx.android.synthetic.main.include_simple_title.*

class Demo020806Activity : BaseActivity() {


    override val layoutId: Int
        get() = R.layout.activity_demo020806

    override fun initView() {
        tv_title!!.text = "可垂直和水平滚动的视图"
    }

    @OnClick(R.id.iv_back)
    fun onViewClicked() {
        finish()
    }
}
